app/composer.json

```diff
@@ -8,7 +8,8 @@
     "license": "gpl2",
     "autoload": {
         "psr-4": {
-            "Silvanusp\\AtoMap\\": "src/"
+            "Silvanusp\\AtoMap\\": "src/",
+            "": "src/"
```

Comment:
this isnt strictly neccessary, if you forget to add a namespoace to one of your classes an its directly in src then this should pick it up. normally however stick to the real namespace for your own classes